src/handlers/me.js

```diff
@@ -19,6 +24,39 @@ module.exports = async function me(member, _, message) {
       )
     }
     if (verificationInfo.userVerified) {
+      // Sponsor member
+      // Check IDChain to see how many sponsorships this address has
+      const provider = new ethers.getDefaultProvider()
+      const contract = new ethers.Contract(
+        CONTRACT_ADDRESS,
+        CONTRACT_ABI,
+        provider,
+      )
+
+      // Get the number of sponsorships assigned to the Discord context by the specified address
+      const formattedContext = ethers.utils.formatBytes32String('Discord')
+      const spBalance = await contract.contextBalance(address, formattedContext)
+      const formattedBalance = parseInt(ethers.utils.formatUnits(spBalance, 0))
+
+      // Assign that many sponsorships in the gist for the guild with this address
+      updateGist(message.guild.id, {
+        assignedSp: formattedBalance,
+      })
+
+      // CONTEXT_ID and ID are a little confusing here
+      // CONTEXT_ID corresponds to the application context that is sponsoring the user, 'Discord' in this case
+      // ID corresponds to the unique ID of the user being sponsored, which is referred to as contextId on BrightID's end
+      if (ethers.utils.formatUnits(spBalance, 0) > 0) {
+        sponsor(sponsorKey, CONTEXT_ID, ID)
+        // Now that the user is sponsored, we need to decrement the amount of sp for this guild
+        updateGist(message.guild.id, {
+          assignedSp: formattedBalance - 1,
+        })
```

Comment:
We might need to check that they weren't already sponsored before deducting a sponsorship.

One way to do this is to call `https://app.brightid.org/node/v5/verifications/Discord/<userId>` (eg. https://app.brightid.org/node/v5/verifications/Discord/e027919a-c141-5e06-abee-8bbbbe024c6a )

If there is a `403`, then they're not sponsored.  See https://dev.brightid.org/docs/node-api/b3A6NDIyNzY4Mw-gets-a-signed-verification#Responses